### ISSUE TITLE
angular cli Error

### ISSUE THREAD

**Issue by CREPAAS** [swimlane/ngx-dnd#3]
Angular CLI 4.0.2 and 4.0.3 

ng serve => error
**ERROR in BrowserAnimationsModule is not an NgModule**

ng serve --aot => error
**ERROR in Unexpected value 'BrowserAnimationsModule in /node_modules/@swimlane/ngx-dnd/release/node_modules/@angular/platform-browser/animations/index.d.ts' imported by the module 'NgxDnDModule in /node_modules/@swimlane/ngx-dnd/release/ngx-dnd.module.d.ts'. Please add a @NgModule annotation.**

What should I do?

```
===================================================
**package.json**

"dependencies": {
    "@angular/animations": "^4.0.2",
    "@angular/common": "^4.0.2",
    "@angular/compiler": "^4.0.2",
    "@angular/core": "^4.0.2",
    "@angular/flex-layout": "^2.0.0-beta.7",
    "@angular/forms": "^4.0.2",
    "@angular/http": "^4.0.2",
    "@angular/platform-browser": "^4.0.2",
    "@angular/platform-browser-dynamic": "^4.0.2",
    "@angular/router": "^4.0.2",
    "core-js": "2.4.1",
    "rxjs": "^5.1.0",
    "zone.js": "^0.8.4",
    "@swimlane/ngx-dnd": "^2.0.1",
    "@swimlane/ngx-ui": "^14.2.4",
    "ng2-cookies": "^1.0.11",
    "ng2-webstorage": "^1.6.2",
    "jquery": "^3.1.1",
    "toastr": "^2.1.2"
  },
  "devDependencies": {
    "@angular/cli": "1.0.0",
    "@angular/compiler-cli": "^4.0.2",
    "@angular/platform-server": "^4.0.2",
    "@types/jasmine": "^2.5.38",
    "@types/node": "^7.0.5",
    "@types/jquery": "^2.0.40",
    "@types/toastr": "^2.1.33",
    "codelyzer": "~2.0.0",
    "jasmine-core": "~2.5.2",
    "jasmine-spec-reporter": "~3.2.0",
    "karma": "~1.4.1",
    "karma-chrome-launcher": "~2.0.0",
    "karma-cli": "~1.0.1",
    "karma-jasmine": "~1.1.0",
    "karma-jasmine-html-reporter": "0.2.2",
    "karma-coverage-istanbul-reporter": "0.2.0",
    "protractor": "~5.1.0",
    "ts-node": "~2.0.0",
    "tslint": "~4.5.0",
    "typescript": "~2.2.0"
  }
```

**Comment by Gerros** [comment]
I encountered the same issue...

**Comment by Gerros** [comment]
workaround:
shift the imports after the declarations in the NgModule configuration

@NgModule({
  imports: [
    ...
    NgxDnDModule,
  ],
  declarations: [
   ...
  ],
  ...
}

to

@NgModule({
  declarations: [
   ...
  ],
  imports: [
    ...
    NgxDnDModule,
  ],
  ...
}

**Comment by cchatfield** [comment]
same error as well.

Workaround did not resolve this issue.

**Comment by Hypercubed** [comment]
Can someone provide a plunker or repo that demonstrates this issue?

**Comment by Hypercubed** [comment]
I published a new version (2.1.2) that removes the `BrowserAnimationsModule` from `NgxDnDModule`.  `BrowserAnimationsModule` will need to be included in your app.  Please open again if the issue persists.

**Comment by cchatfield** [comment]
I was just working on the bug repo to reproduce and I was having challenges reproducing until I noticed the new release. Thanks for the release to resolve.

I was able to fix my local on the prior release by replacing the @swimlane/ngx-dnd/node_modules/@angular/platform-browser with the @angular 4.1.3 node_modules files.
